boards/nxp/imx943_evk/doc/index.rst

```diff
@@ -69,6 +69,11 @@ NETC driver supports to manage the Physical Station Interface (PSI).
 The ENET0, ENETC1, ENETC2 ports could be enabled for M33 by west build option
 ``-DEXTRA_DTC_OVERLAY_FILE=enetc.overlay``.
 
+I2C
+---
+
+Enable lpi2c6 defaultly for M33(In NETC MIX).
+
```

Comment:
not grammatically correct and doesn't need to go in the doc page, its already captured on supported feature table which is generated